puppet/zulip/files/supervisor/conf.d/zulip_postsetup.conf

```diff
@@ -0,0 +1,9 @@
+; Used in dockervoyager.pp, runs the postsetup script that creates a user in Zulip
+; Only run for Docker environments if not disabled by the entrypoint script
+[program:zulip-postsetup-create_user]
+command = bash -c "$("$(realpath $(dirname $0)/../..)"/scripts/lib/zulip_tools.py make_deploy_path)/scripts/create-zulip-admin.sh"
+user=root
+stdout_events_enabled=true
+stderr_events_enabled=true
+autorestart = false
+exitcodes = 0,1,127
```

Comment:
I don't understand why this command, which presumably should only be run once, is a supervisord config file.

Reply:
(summoning @galexrt to provide the rationale)

Reply:
I have just removed the run-only-once postsetup and rebased. @timabbott anything else needs to be fixed / rationalized?